var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -17,102 +22,58 @@ class Graphviz(AutotoolsPackage):
-            ' (not yet functional)')
-    variant('go', default=False,
-            description='Enable for optional go language bindings'
-            ' (not yet functional)')
-    variant('guile', default=False,
-            description='Enable for optional guile language bindings'
-            ' (not yet functional)')
-    variant('io', default=False,
-            description='Enable for optional io language bindings'
-            ' (not yet functional)')
+    # Language bindings
     variant('java', default=False,  # Spack has no Java support
             description='Enable for optional java language bindings')
-    variant('lua', default=False,
-            description='Enable for optional lua language bindings'
-            ' (not yet functional)')
-    variant('ocaml', default=False,
-            description='Enable for optional ocaml language bindings'
-            ' (not yet functional)')
-    variant('perl', default=False,    # Spack has no Perl support
-            description='Enable for optional perl language bindings')
-    variant('php', default=False,
-            description='Enable for optional php language bindings'
-            ' (not yet functional)')
-    variant('python', default=False,    # Build issues with Python 2/3
-            description='Enable for optional python language bindings'
-            ' (not yet functional)')
-    variant('r', default=False,
-            description='Enable for optional r language bindings'
-            ' (not yet functional)')
-    variant('ruby', default=False,
-            description='Enable for optional ruby language bindings'
-            ' (not yet functional)')
-    variant('tcl', default=False,
-            description='Enable for optional tcl language bindings'
-            ' (not yet functional)')
 
-    variant('pangocairo', default=False,
-            description='Build with pango+cairo support (more output formats)')
-    variant('libgd', default=False,
-            description='Build with libgd support (more output formats)')
-    variant('gts', default=False,
-            description='Build with GNU Triangulated Surface Library')
+    # Feature bindings
     variant('expat', default=False,
             description='Build with Expat support (enables HTML-like labels)')
+    variant('gts', default=False,
+            description='Build with GNU Triangulated Surface Library')
     variant('ghostscript', default=False,
             description='Build with Ghostscript support')
-    variant('qt', default=False,
-            description='Build with Qt support')
     variant('gtkplus', default=False,
             description='Build with GTK+ support')
-
-    parallel = False
-
-    # These language bindings have been tested, we know they work.
-    tested_bindings = ('+java', )
-
-    # These language bindings have not yet been tested.  They
-    # likely need additional dependencies to get working.
-    untested_bindings = (
```

Comment:
@sethrj I agree with you that the untested bindings shouldn't be actual variants, and I also agree with @citibeth that these should not be removed entirely so that future package contributors could know what is available: would you mind keeping this list?

Reply:
I have restored the untested binding options but have disabled them as variants for the time being.